<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>920</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>920</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>920</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">KOMI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	color: rgb(190, 190, 255);
	background-color: rgb(75, 111, 166);
	border: 4px solid black;
	border-radius: 10px;
}

QPushButton:hover
{
	color: rgb(225, 225, 225);
	background-color: rgb(0, 160, 117);
	border: 3px solid black;
}

QPushButton:pressed
{
	color: rgb(195, 195, 195);
	background-color: rgb(0, 130, 87);
	border: 5px solid black;
}

QTextEdit
{
	border: 5px solid rgb(0, 0, 0);
	border-radius: 10px
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QGridLayout" name="grid"/>
   </widget>
   <widget class="QPushButton" name="btnCreate">
    <property name="geometry">
     <rect>
      <x>1196</x>
      <y>66</y>
      <width>323</width>
      <height>114</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Okta Neue</family>
      <pointsize>8</pointsize>
      <weight>1</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255,0);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:hover
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(107, 189, 197,111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:pressed
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(62, 96, 116, 111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}</string>
    </property>
    <property name="text">
     <string>Заполнить матрицу!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>1517</x>
      <y>178</y>
      <width>323</width>
      <height>115</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Okta Neue</family>
      <pointsize>8</pointsize>
      <weight>1</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255,0);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:hover
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(107, 189, 197,111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:pressed
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(62, 96, 116, 111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}</string>
    </property>
    <property name="text">
     <string>Добавить вершину!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>1196</x>
      <y>291</y>
      <width>323</width>
      <height>114</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Okta Neue</family>
      <pointsize>8</pointsize>
      <weight>1</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255,0);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:hover
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(107, 189, 197,111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:pressed
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(62, 96, 116, 111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}</string>
    </property>
    <property name="text">
     <string>Удалить вершину!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnswer">
    <property name="geometry">
     <rect>
      <x>1514</x>
      <y>644</y>
      <width>323</width>
      <height>114</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Okta Neue</family>
      <pointsize>8</pointsize>
      <weight>1</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255,0);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:hover
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(107, 189, 197,111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:pressed
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(62, 96, 116, 111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}</string>
    </property>
    <property name="text">
     <string>Рассчитать расстояние!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>1517</x>
      <y>404</y>
      <width>323</width>
      <height>114</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Okta Neue</family>
      <pointsize>8</pointsize>
      <weight>1</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255,0);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:hover
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(107, 189, 197,111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}


QPushButton:pressed
{font: 12 8pt &quot;Okta Neue&quot;;
border-color: rgba(255, 255, 255, 0);
	background-color: rgba(62, 96, 116, 111);
color: rgba(255, 255, 255, 0);
border-radius: 57px;
}</string>
    </property>
    <property name="text">
     <string>Изменить путь!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lengthText">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>820</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Okta Neue</family>
      <pointsize>19</pointsize>
      <weight>1</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 12 19pt &quot;Okta Neue&quot;;
color: rgb(1, 0, 21);
border-color:rgba(0,0,0,0);
border-radius:30px;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="wayText">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>820</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Okta Neue</family>
      <pointsize>15</pointsize>
      <weight>1</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(1, 0, 21);
border-color:rgba(0,0,0,0);
border-radius:30px;
font: 12 15pt &quot;Okta Neue&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-20</y>
      <width>1920</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/s/коми3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="rootText">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>586</y>
      <width>321</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border-radius:28px;
font: 12 18pt &quot;Okta Neue&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhMultiLine</set>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>gridWidget</zorder>
   <zorder>btnCreate</zorder>
   <zorder>btnAdd</zorder>
   <zorder>btnDelete</zorder>
   <zorder>btnAnswer</zorder>
   <zorder>btnEdit</zorder>
   <zorder>lengthText</zorder>
   <zorder>wayText</zorder>
   <zorder>rootText</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
